resources/packaging/opensuse/ungoogled-chromium.spec.in

```diff
@@ -0,0 +1,549 @@
+BuildRequires:  pkgconfig(vdpau)
+BuildRequires:  pkgconfig(vorbis)
+BuildRequires:  pkgconfig(x11)
+BuildRequires:  pkgconfig(xcomposite)
+BuildRequires:  pkgconfig(xcursor)
+BuildRequires:  pkgconfig(xdamage)
+BuildRequires:  pkgconfig(xext)
+BuildRequires:  pkgconfig(xfixes)
+BuildRequires:  pkgconfig(xi)
+BuildRequires:  pkgconfig(xrandr)
+BuildRequires:  pkgconfig(xrender)
+BuildRequires:  pkgconfig(xscrnsaver)
+BuildRequires:  pkgconfig(xt)
+BuildRequires:  pkgconfig(xtst)
+BuildRequires:  pkgconfig(zlib)
+Requires:       hicolor-icon-theme
+Requires:       xdg-utils
+Requires(pre):  permissions
+Recommends:     noto-coloremoji-fonts
+Recommends:     noto-emoji-fonts
+Conflicts:      chromium-browser
+Provides:       chromium-based-browser = %{version}
+Provides:       chromium-browser = %{version}
+Obsoletes:      %{name}-suid-helper < %{version}
+Obsoletes:      chromium-browser < %{version}
+Provides:       %{name}-suid-helper = %{version}
+Obsoletes:      chromium-beta-desktop-gnome
+Obsoletes:      chromium-beta-desktop-kde
+Obsoletes:      chromium-desktop-gnome
+Obsoletes:      chromium-desktop-kde
+Obsoletes:      chromium-dev-desktop-gnome
+Obsoletes:      chromium-dev-desktop-kde
+Obsoletes:      chromium-ffmpeg
+Obsoletes:      chromium-ffmpegsumo
+# no 32bit supported and it takes ages to build
+ExcludeArch:    %{arm} aarch64 i586 ppc ppc64 ppc64le
+%if 0%{?suse_version} >= 1330
+BuildRequires:  nodejs8
+%else
+BuildRequires:  nodejs6
+%endif
+%if %{with system_libxml}
+BuildRequires:  pkgconfig(libxml-2.0) >= 2.9.5
+%endif
+%if !%{with sle_bundles}
+BuildRequires:  python-beautifulsoup4
+BuildRequires:  python-html5lib
+BuildRequires:  python-simplejson
+BuildRequires:  python-xml
+BuildRequires:  yasm-devel
+%endif
+%if %{with system_icu}
+BuildRequires:  pkgconfig(icu-i18n) >= 59.0
+%endif
+%if %{with system_vpx}
+BuildRequires:  pkgconfig(vpx) >= 1.6.1
+%endif
+BuildRequires:  clang >= 3.9.0
+
+%description
+Chromium is the open-source project behind Google Chrome. We invite you to join us in our effort to help build a safer, faster, and more stable way for all Internet users to experience the web, and to create a powerful platform for developing a new generation of web applications.
+
+%package -n chromedriver
+Summary:        WebDriver for Google Chrome/Chromium
+License:        BSD-3-Clause
+Group:          Development/Tools/Other
+Requires:       %{name} = %{version}
+
+%description -n chromedriver
+WebDriver is an open source tool for automated testing of webapps across many browsers. It provides capabilities for navigating to web pages, user input, JavaScript execution, and more. ChromeDriver is a standalone server which implements WebDriver's wire protocol for Chromium. It is being developed by members of the Chromium and WebDriver teams.
+
+
+# Setup for build 
+%prep
+#%setup -q -n %{rname}-%{version}
+cd %{_topdir}/BUILD
+rm -rf ungoogled-chromium-%{version}
+mv $ungoog{unpacked_src_dir} .
```

Comment:
Instead of manipulating the buildspace tree here, it's better to do it in `ungoogled_packaging.sh`. You can move the buildspace tree at the end of that script, or just add a hardlink or softlink to the buildspace tree from `BUILD/`. That way, we can remove the hardcoded dependency on `unpacked_src_dir`.